apps/swirl-docs/src/pages/icons/components/IconGrid.tsx

```diff
@@ -0,0 +1,36 @@
+import { FunctionComponent } from "react";
+import { IconsMetaData } from "..";
+
+interface IconGridProps {
+  iconList: string[];
+  icons: IconsMetaData;
+  handleTileClick: (iconName: string) => void;
+}
+
+export const IconGrid: FunctionComponent<IconGridProps> = ({
+  iconList,
+  icons,
+  handleTileClick,
+}) => {
+  return (
+    <ul className="grid grid-cols-2 md:grid-cols-fill-rows gap-4 w-full">
+      {iconList?.map((icon: string, index: number) => (
+        <li key={`${icons[icon]?.name}-${index}`}>
```

Comment:
These create a lot of tab stops. I'm not 100% sure about it, but implementing it as a LayoutGrid (https://www.w3.org/WAI/ARIA/apg/example-index/grid/LayoutGrids.html) with arrow key navigation might be better.

Reply:
Yes this is totally true. It would be better. I will take a look into this